graph-server/Dockerfile

```diff
@@ -0,0 +1,44 @@
+# Build the static assets in the node.js container.
+FROM wdfn_assets as assets
+RUN npm run build
+
+# Use LTS release
+FROM node:10.15-alpine
+
+# Installs latest Chromium package.
+RUN apk update && apk upgrade && \
+    echo @edge http://nl.alpinelinux.org/alpine/edge/community >> /etc/apk/repositories && \
```

Comment:
~I see we're pinning to Alpine's edge repository. Should we be concerned about any of these packages changing underneath our feet without us knowing about it?~

Reply:
Nevermind. You want to latest package.